README.md

```diff
@@ -1,59 +1,85 @@
 Salary slip kata
 ================
-### Problem description: Salary slip generator for UK companies.
- 
-  A typical salary slip contains employee details like employee id, employee name and their salary details like their gross salary, national insurance contributions , taxable income, tax-free allowance* and taxes payable.
-  
-  In the event there are earnings like overtime and bonuses or deductions like loans, these are accounted for in the respective aspects of the salary slip and have an impact on the gross salary, national insurance contributions, and taxes to be paid.
-  
+## Problem description: Salary slip generator for UK companies.
+
+  A typical salary slip contains employee details like employee id, employee name and their salary details like their gross salary, national insurance contributions, taxable income, tax-free allowance and taxes payable.
```

Comment:
Just noticed its best to maintain the order of the terms tax-free allowance, taxable income and tax payable in our sentences throughout the description, this stems from the original document, I'll change it there to make it consistent - thanks